servicenotifier/doc/adoc/index.adoc

```diff
@@ -1,3 +0,0 @@
-= ServiceNotifier
-
-* xref:servicenotifier.adoc#[ServiceNotifier API]
```

Comment:
Bitte noch einen Verweis auf die verschobene Adoc-Datei in der index.adoc hinzufügen.

Reply:
Erledigt.